install-dev/data/xml/tab.xml

```diff
@@ -5,450 +5,697 @@
         <field name="class_name"/>
         <field name="active"/>
         <field name="icon"/>
+        <field name="wording"/>
+        <field name="wording_domain"/>
     </fields>
     <entities>
 
         <tab id="Dashboard" id_parent="" active="1" enabled="1" hide_host_mode="0" icon="trending_up">
             <class_name>AdminDashboard</class_name>
+            <wording>Dashboard</wording>
+            <wording_domain>Admin.Navigation.Menu</wording_domain>
         </tab>
 
         <!--SELL-->
         <tab id="SELL" id_parent="" active="1" enabled="1" hide_host_mode="0" icon="">
             <class_name>SELL</class_name>
```

Comment:
Is these  `class_name` and `id` are ok? :thinking: 
```suggestion
        <tab id="Sell" id_parent="" active="1" enabled="1" hide_host_mode="0" icon="">
            <class_name>Sell</class_name>
```

Reply:
I don't remember what I found out about these but since it's unrelated I'd prefer to leave it as is for now. Be aware that this tab exists as it is since 1.7.0.